lib/registerTypes/RegisterTypes.h

```diff
@@ -140,11 +140,11 @@ void registerTypesMapObjectTypes(Serializer &s)
 	s.template registerType<IUpdater, GrowsWithLevelUpdater>();
 	s.template registerType<IUpdater, TimesHeroLevelUpdater>();
 	s.template registerType<IUpdater, TimesStackLevelUpdater>();
+	s.template registerType<IUpdater, OwnerUpdater>();

```

Comment:
Are you sure it will not break serialization?

Reply:
It will. Good catch. I'll move this commit into new-terrains feature branch where we don't have backward compatibility anyway